moto/organizations/responses.py

```diff
@@ -115,6 +120,11 @@ def list_policies(self):
             self.organizations_backend.list_policies(**self.request_params)
         )
 
+    def delete_policy(self):
+        return json.dumps(
```

Comment:
The backend-method doesn't seem to return anything. It probably reads better to rewrite this to:
```
self.organizations_backend.delete_policy(**self.request_params)
return json.dumps({})
```